### ISSUE TITLE
Hardcoded Values

### ISSUE THREAD

**Issue by MurderAxolotl** [paulgrey666/Moderna-Conky#2]
You use hardcoded locations in your start script (`/home/paulgrey/Downloads/ConkyBar/Custom_Clock/Data/conky_cpu`, for example)

This will not work for other users; you should instead detect the script's starting location.

As per [this Stackoverflow post](https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script), you can grab the script's location and put it in a variable using `ScriptLocation=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )`

Then simply use that script location instead of hardcoded paths, like so:
`/home/paulgrey/Downloads/ConkyBar/Custom_Clock/Data/conky_cpu` would become `$ScriptLocation/conky_cpu`

Not only does this allow the script to be run from anywhere, 

DO NOTE: You have hardcoded values in all sorts of places in your project here.
For example, you try loading `/home/paulgrey/Downloads/ConkyBar/Custom_Clock/clock_rings.lua` in your *rings* file,. There are similar issues in `/Data/conky_files`, `conky_system`

Additional note: the file you provide, the `tar.gz` archive, does not contain a `scripts` folder. This leads to `uptime.sh` not being found

One last note: You shouldn't put your source code into an archive on GitHub. The whole point is to upload your files individually, like [this example](https://github.com/vinceliuice/Graphite-gtk-theme). Use releases for compressed and ready-to-distribute versions of your code

IN SUMMARY:
- Don't use hard-coded values
- Ship *all* the files needed to make your project functional
- Don't compress source code on GitHub
- Use releases
